src/ray/protobuf/export_api/export_train_state.proto

```diff
@@ -62,6 +62,19 @@ message ExportTrainRunEventData {
   optional string controller_log_file_path = 10;
   // Version of Ray Train
   uint32 ray_train_version = 11;
+
+  // Dashboard panel configuration for visualizing Train run data
+  message DashboardPanel {
+    // Unique identifier for the panel
+    string id = 1;
+    // Display name of the panel
+    string title = 2;
+  }
+
+  // List of TrainRun dashboard panels
+  repeated DashboardPanel train_run_panels = 12;
+  // List of dashboard panels to include for TrainWorkers
```

Comment:
let's include instructions here in the comments that these panels should be scoped to run_id, worker_id, etc